packages/icons/src/metadata.ts

```diff
@@ -0,0 +1,1816 @@
+/**
+ * Metadata for all available onyx icons.
+ */
```

Comment:
[opinion] I'd prefer to split the file and move the ICON_METADATA to a separate file from the rest of the file below it. for our own DX's sake when we want to work on the things going on below ^^ this section looks kinda auto generated